src/main/kotlin/net/atos/zac/admin/ZaakafhandelParameterBeheerService.kt

```diff
@@ -178,13 +182,13 @@ class ZaakafhandelParameterBeheerService @Inject constructor(
     /**
      * Kopieren van de HumanTaskParameters van de oude ZaakafhandelParameters naar de nieuw ZaakafhandelParameters
      *
-     * @param vorigeZaakafhandelparameters bron
-     * @param nieuweZaakafhandelParameters bestemming
+     * @param previousZaakafhandelParameters bron
+     * @param newZaakafhandelParameters bestemming
      */
     private fun mapHumanTaskParameters(
-        vorigeZaakafhandelparameters: ZaakafhandelParameters,
-        nieuweZaakafhandelParameters: ZaakafhandelParameters
-    ) = vorigeZaakafhandelparameters.humanTaskParametersCollection.map {
+        previousZaakafhandelParameters: ZaakafhandelParameters,
+        newZaakafhandelParameters: ZaakafhandelParameters
+    ) = previousZaakafhandelParameters.humanTaskParametersCollection.map {
```

Comment:
mapping entities can be done easier, and per usage in the other method, I'd go for this:
```kotlin
    private fun Set<HumanTaskParameters>.clone() = map {
            it.id = null // detach the entity
            it.zaakafhandelParameters = null
            it
        }
        .toSet()
```